src/api/app/components/notification_notifiable_link_component.rb

```diff
@@ -110,6 +114,8 @@ def notifiable_link_path
       else
         Rails.application.routes.url_helpers.project_users_path(@notification.event_payload['project'], notification_id: @notification.id)
       end
+    when 'Event::AddedUserToGroup', 'Event::RemovedUserFromGroup'
+      nil
```

Comment:
Link to the group show page?

Reply:
The group show page is only accessible to admins and maintainers. It goes through this policy method:

https://github.com/openSUSE/open-build-service/blob/547b7fafa89b38ed22dc216f4d8dd31e2b70021e/src/api/app/policies/group_policy.rb#L12

So for the majority of users there won't be any link, so I decided not to add it. Admins and maintainers can easily reach the group in case they want to.

Reply:
You're right! I don't know what failed on my tests that I couldn't access and assumed the policy was the limitation.... :thinking: 

Alright, I'm going to add the link.

Reply:
Done.

![Screenshot 2024-06-12 at 13-08-19 Open Build Service](https://github.com/openSUSE/open-build-service/assets/2581944/f1d01294-9f42-4d9f-ac09-5fa12b9faeca)